sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobContainerItemProperties.java

```diff
@@ -348,20 +348,18 @@ public BlobContainerItemProperties setRemainingRetentionDays(Integer remainingRe
     }
 
     /**
-     * Get the isImmutableStorageWithVersioningEnabled property: Indicates if version level worm is enabled on this
-     * container.
+     * Get the isVersionLevelWormEnabled property: Indicates if version level worm is enabled on this container.
      *
-     * @return the isImmutableStorageWithVersioningEnabled value.
+     * @return the isVersionLevelWormEnabled value.
```

Comment:
did you accidentally manually change this during the merge? Can we change it back?

Reply:
same for below as well